scripts/order.json

```diff
@@ -24,6 +24,19 @@
         { "name": "submitted_at", "type": "date-time-iso8601" },
         { "name": "total", "type": "decimal" }
       ]
+    },
+    "orderform": {
+      "fields": [
+        { "name": "merchant_id", "type": "string" },
+        { "name": "total", "type": "decimal" }
+      ]
+    },
+    "error": {
+      "fields":[
+        { "name": "code", "type": "string" },
+        { "name": "message", "type": "string" },
+        { "name": "details", "type": "string" }
```

Comment:
no need for `details`. All the information can be stored in `message`. Also, let's make message a sequence of string.